templates/scenarios/csharp/non-sso-tab/env/.env.dev.user.tpl

```diff
@@ -0,0 +1 @@
+# Secrets. Keys prefixed with `SECRET_` will be masked in Teams Toolkit logs.
```

Comment:
If the project does not save any content to .env.xxx.user, my personal opinion is remove the `.env.xxx.user` file from template.
@jidddddd what's your opinion?

Reply:
But we still need to ignore the .user file incase user adds it manually

Reply:
@blackchoey yes we can remove the file from template